### PR TITLE
Updated mailchimp form to be email octopus

### PR DESCRIPTION
**Pull request by Oceas**
Updated newsletter signup form to use email octopus replacing Mailchimp.
Issue: https://github.com/WebDevStudios/custom-post-type-ui/issues/776

About Newsletter Signup
Terms/Taxonomy Newsletter Signup

### CODE REVIEW

File: css/cptui.scss [hunk 1]
```diff
@@ -200,13 +200,20 @@ $max-tablet: "screen and (max-width: 768px)";
...
 	}
-	.cptui-badge {
-		position: absolute;
-	    top: 0;
-	    right: 0;
+}
+.email-octopus-form-row {
+	input::placeholder {
+		color: #cccccc;
 	}
```
Comment (tw2113) [inline css/cptui.scss]: Let's keep the cptui-badge selector as it puts the graphic in the upper right corner.

Comment (tw2113) [reply css/cptui.scss]: Ah, I'm seeing some markup changes below related to this. If I recall right, I probably borrowed heavily from BuddyPress/bbPress for layout for this.

Still would be interesting to see about alignment options.

File: inc/utility.php [hunk 2]
```diff
@@ -369,37 +369,52 @@ function cptui_products_sidebar() {
...
+    <h2 class="email-octopus-heading">Subscribe to Pluginize.com</h2>
+    <p class="email-octopus-success-message"></p>
+    <p class="email-octopus-error-message"></p>
+
+    <form method="post"
+      action="https://emailoctopus.com/lists/2039e001-4775-11ea-be00-06b4694bee2a/members/embedded/1.3/add"
+      class="email-octopus-form"
+      data-sitekey="6LdYsmsUAAAAAPXVTt-ovRsPIJ_IVhvYBBhGvRV6"
+    >
+
```
Comment (salcode) [inline inc/utility.php]: Though not visible here on GitHub, there is a trailing space on this line. Could we please remove it?

Note: I'm sure this came along with the code we received to add (but I'd still like to correct it).

File: inc/utility.php [hunk 3]
```diff
@@ -369,37 +369,52 @@ function cptui_products_sidebar() {
...
+        <div class="email-octopus-form-row-subscribe">
+            <input type="hidden"
+               name="successRedirectUrl"
+               value="">
+            <button type="submit">Subscribe</button>
+        </div>
+    </form>
+
+
```
Comment (salcode) [inline inc/utility.php]: Though not visible here on GitHub, there is a trailing space on this line. Could we please remove this line entirely.

Note: I'm sure this came along with the code we received to add (but I'd still like to correct it).

File: inc/utility.php [hunk 4]
```diff
@@ -369,37 +369,52 @@ function cptui_products_sidebar() {
...
-			<div style="position: absolute; left: -5000px;" aria-hidden="true">
-				<input type="text" name="b_67169b098c99de702c897d63e_9cb1c7472e" tabindex="-1" value=""></div>
-			<div class="clear">
-				<input type="submit" value="<?php esc_attr_e( 'Subscribe', 'custom-post-type-ui' ); ?>" name="subscribe" id="mc-embedded-subscribe" class="button" tabindex="-1">
-			</div>
-		</div>
-	</form>
+
+<div class="email-octopus-form-wrapper">
+    <h2 class="email-octopus-heading">Subscribe to Pluginize.com</h2>
```
Comment (salcode) [inline inc/utility.php]: Here is a screenshot comparing the previous version with the updated version of the Promo signup.

The previous version appears on the left, the updated version on the right.

Screen Shot 2020-02-04 at 16 05 09

Despite the fact we were given the markup as a block, I don't see any reason we can't restore the text to 

> Get email updates from pluginize.com about Custom Post Type UI

unless this text update was a specific branding change (which I suspect it was not, but I could be wrong).

cc: @tw2113

File: inc/about.php [hunk 5]
```diff
@@ -157,9 +157,13 @@ function cptui_about_page_newsletter() {
 	}
 
 	?>
-	<h3><?php esc_html_e( 'Stay informed', 'custom-post-type-ui' ); ?></h3>
```
Comment (salcode) [inline inc/about.php]: Here is a screenshot comparing the old version (on the top) with the new version (on the bottom).

- The "Stay informed" text has been removed
- The cptui-badge now sits lower than it did previously

Screen Shot 2020-02-04 at 16 14 34

File: inc/about.php [hunk 6]
```diff
@@ -157,9 +157,13 @@ function cptui_about_page_newsletter() {
...
+	<div class='wdsoctosignup'>
+		<?php
+		cptui_newsletter_form();
```
Comment (salcode) [inline inc/about.php]: Here is a screenshot comparing the old version (on the top) with the new version (on the bottom).

- the old version did not have the white background
- the styling of the old `Subscribe` button was significantly different

Screen Shot 2020-02-04 at 16 17 38

Comment (Oceas) [reply inc/about.php]: These design changes were intentional to better match the rest of the page.

File: inc/utility.php (hunk 4 above)
Comment (Oceas) [reply inc/utility.php]: The text can absolutely be set back to the original. However, the gist Brad had provided had the updated title text, felt safer to carry the change over.

Comment (Oceas) [reply inc/utility.php]: Went ahead and updated header.

**Pull request merged**